cloud/google/cmd/gce-machine-controller/README.md

```diff
@@ -0,0 +1,27 @@
+# GCE Machine Controller
+
+The GCE Machine Controller is a machine controller implementation for clusters running on Google Compute Engine.
+
+## Development
+The instructions below will enable you to get started with running your own version of the gce-machine-controller on a GCE Kubernetes cluster.
+
+### Prerequisites
+
+1. Follow the "Before you begin" section from the google cloud container registry [pushing and pulling images documentation](https://cloud.google.com/container-registry/docs/pushing-and-pulling).
+1. From this directory, run `make dev_push`.
+1. Note from the ouput of the above step the name of the image. It should be in this format, `gcr.io/my-gcp-project-name/gce-machine-controller:0.0.7-dev`.
```

Comment:
Might also make sense to sub out the 0.0.7 with [VERSION] or something like that

Reply:
Yeah, I was a bit worried about confusing people, but I'll change to VERSION as I think you are right.